packages/apps/proxy/src/middleware/cache/readThroughCacheMiddleware.ts

```diff
@@ -78,10 +82,6 @@ export default async ({ proxyTarget }: { proxyTarget: string }) => {
         res.status(500).json({ message: "Proxy error" });
       },
       logLevel: "silent",
-      followRedirects: true,
-      ...(process.env.NODE_TLS_REJECT_UNAUTHORIZED === "0"
-        ? { secure: false }
-        : {}),
```

Comment:
Is this intentional?  I think this was required to make enterprise proxies work and enable easier debugging